lib/telemetry_loggers/live_view_logger.ex

```diff
@@ -0,0 +1,93 @@
+defmodule TelemetryLoggers.LiveViewLogger do
+  @moduledoc """
+  Logs one log per request on a LiveView call.
+
+  It includes the following metadata:
+  - `callback` - Which kind of live view event is listening on, eg: `:mount`, `:handle_params`, etc...
+  - `kind` - The source of the event, eg: `live_view` or `live_component`
+  - `duration`: The duration in microseconds
+
+  To set this up properly, you want to disable Phoenix built-in logger by adding
+  to your config:
+
+      config :phoenix, :logger, false
```

Comment:
Do the phoenix default logger log these information? If not, maybe we should not say anything about this in the docs here.